Apps/WebClient/src/ClientApp/src/components/calendar/body.vue

```diff
@@ -207,7 +219,7 @@ export default class CalendarBodyComponent extends Vue {
                 let dayEvent = {
                     id: startDate.getTime().toString() + "-" + perDay,
                     monthDay: startDate.getDate(),
-                    isToday: now.toDateString() == startDate.toDateString(),
+                    isToday: today.toDateString() == startDate.toDateString(),
```

Comment:
mbe unnecessary conversion from datetime to string for comparison? Is it better to have a date time comparison in our util/common?

Reply:
Not sure why this was necessary. I will take a look at it.